FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/SSL Pinning/SSLPinningTests.swift

```diff
@@ -87,7 +87,7 @@ class SSLPinningTests: FRAuthBaseTest {
         //  Init SDK
         self.startSDK()
         
-        let request = Request(url: "https://apple.com", method: .GET)
+        let request = Request(url: "https://webhook.site/309b8710-635b-40d6-9d90-dfbabeef954e", method: .GET)
```

Comment:
Is this request URL something that will always be there or is it ephemeral causing the tests to break in the future?

Reply:
Yes you are right. This was a tempoarary URL that would expire in 7 days. Changed to a permanent one